nym-vpn-app/src/types/tauri.ts

```diff
@@ -156,8 +155,6 @@ export type GatewaysByCountry = {
   quic: boolean;
 };
 
-export type GpuType = 'nvidia' | 'amd' | 'intel' | { unknown: string | null };
-
```

Comment:
same as above

Reply:
(*the above message, idk why but gh seems mixing comments ordering) Please revert that removal, this is due to platform gated defined type in Rust side, and you are developing using a non Linux machine.
The source type will need a bit of rework.
For now, it's OK to keep it as is.